tutor/specs/acceptance/.eslintrc.js

```diff
@@ -1,6 +1,8 @@
 module.exports = {
   "globals": {
     "React":      false,
+    "beforeAll":  false,
+    "afterAll":   false,
     "beforeEach": false,
     "afterEach":  false,
     "describe":   false,
```

Comment:
eslint now allows shared configs that can extend one another, we should probably use that for the specs to prevent all this duplication. 😭